src/miner.cpp

```diff
@@ -329,8 +322,27 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
         LogPrintf("CreateNewBlock(): total size %u\n", nBlockSize);
 
         // Compute final coinbase transaction.
-        txNew.vout[0].nValue = nFees + GetBlockSubsidy(nHeight, chainparams.GetConsensus());
+        CMutableTransaction txNew;
+        txNew.vin.resize(1);
+        txNew.vin[0].prevout.SetNull();
         txNew.vin[0].scriptSig = CScript() << nHeight << OP_0;
+
+        if (nHeight < FOUNDERS_REWARD_UNTIL_BLOCK) {
```

Comment:
This should be the same condition as https://github.com/zcash/zcash/pull/831/commits/11483859b56142b391d4fa89d84a25e51d073568#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR3010 (possibly it doesn't matter if we don't use this code to mine the genesis block, but it's confusing).